apps/web/src/hooks/wallets/consts.ts

```diff
@@ -16,7 +14,6 @@ export const CGW_NAMES: { [_key in WALLET_KEYS]: string | undefined } = {
   [WALLET_KEYS.WALLETCONNECT_V2]: 'walletConnect_v2',
   [WALLET_KEYS.COINBASE]: 'coinbase',
   [WALLET_KEYS.LEDGER]: 'ledger',
```

Comment:
Should Ledger (v1) be removed?

Reply:
I removed all reference and logic related to the previous implementation as this replaces it. Do you think we should keep it instead?